docker.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
Drop this helper script. Use our `cloudposse/build-harness` instead

Reply:
See `cloudposse/geodesic` for an example.

Add the `Makefile`. Then just run `make docker/build`

Reply:
done